tests/utils.ts

```diff
@@ -8,9 +8,29 @@ import { fileURLToPath } from 'node:url';
 const __dirname = path.dirname(fileURLToPath(import.meta.url));
 
 const blueprintPath = path.join(__dirname, '..');
+const fixturesPath = path.join(__dirname, 'fixtures');
 
 export const SUPPORTED_PACKAGE_MANAGERS = ['npm', 'yarn', 'pnpm'] as const;
 
+export async function fixture(
+  /**
+   * Which file within in the fixture-set / scenario to read
+   */
+  file: string,
+  options?: {
+    /**
+     * Which fixture set to use
+     */
+    scenario?: string;
+  }
+) {
+  let scenario = options?.scenario ?? 'default';
```

Comment:
I don't like that the `default` is the default value for scenario here, because it matters what folder you put the test file in. I feel like you could make this just a plain argument of the function and make it required, rather than part of the options and making it optional

Reply:
FWIW, I don't find `default` being the default too surprising 🙃